src/odemis/gui/cont/correlation.py

```diff
@@ -0,0 +1,294 @@
+
+# -*- coding: utf-8 -*-
+
+"""
+@author Patrick Cleeve
+
+Copyright © 2023, Delmic
+
+Handles the controls for correlating two (or more) streams together.
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the
+terms of the GNU General Public License version 2 as published by the Free
+Software Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A
+PARTICULAR PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+
+"""
+import logging
+import math
+import wx
+# IMPORTANT: wx.html needs to be imported for the HTMLWindow defined in the XRC
+# file to be correctly identified. See: http://trac.wxwidgets.org/ticket/3626
+# This is not related to any particular wxPython version and is most likely permanent.
+import wx.html
+import odemis.acq.stream as acqstream
+import odemis.gui.model as guimod
+from odemis import model
+from odemis.acq.stream import StaticStream
+from odemis.gui.util import call_in_wx_main
+
+# TODO: move to more approprate location
+def update_image_in_views(s: StaticStream, views: list) -> None:
+    """Force update the static stream in the selected views
+    param s: (StaticStream) the static stream to update
+    param views: (list[View]) the list of views to update"""
```

Comment:
Can you use `:param [name]:` in all docstrings, you do it in most of them, but missed a few.
```suggestion
    :param s: (StaticStream) the static stream to update
    :param views: (list[View]) the list of views to update"""
```

Reply:
I think i've updated them all now